app/src/components/Layers/TrackLayer.jsx

```diff
@@ -150,7 +133,7 @@ const createTrackLayer = function (google) {
    * @param filters
    * @param vesselTrackDisplayMode
    */
-  TrackLayer.prototype.drawTile = function (data, series, filters, vesselTrackDisplayMode) {
+  TrackLayer.prototype.drawTile = function (data, series, filters, vesselTrackDisplayMode, weight, zoom) {
```

Comment:
Performance could be optimized here by:
- using canvas.Path2D to batch canvas commands (only supported by FF and Chrome, has a polyfill)
- inline function calls (at the cost of code readabilty, see if we can automate this with the transpiler)